cfg/conf.ini.sample

```diff
@@ -75,6 +75,10 @@ languageselection = false
 ; sha256 in HMAC for the deletion token
 zerobincompatibility = false
 
+; allows you to specify the name of the web server you are using to use ParseBin.
```

Comment:
Typo: "ParseBin"

Also a funny name, but was not [our favourite](https://github.com/elrido/ZeroBin/issues/79#issuecomment-242116386)… :wink: